packages/analytics/amplify_analytics_pinpoint/README.md

```diff
@@ -1,6 +1,6 @@
 # amplify_analytics_pinpoint
 
-The Amplify Flutter Analytics category plugin using the AWS Pinpoint provider.
+The Amplify Flutter Analytics category plugin using Pinpoint.
```

Comment:
Why are we making this change?

Reply:
Thanks I changed it back.  I might have copied but not the entire text.